spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/subscriber/PubSubSubscriberOperations.java

```diff
@@ -86,7 +86,8 @@
 	 * Pull and auto-acknowledge a number of messages from a Google Cloud Pub/Sub subscription.
 	 * @param subscription canonical subscription name, e.g., "subscriptionName", or the fully-qualified
 	 * subscription name in the {@code projects/<project_name>/subscriptions/<subscription_name>} format
-	 * @param maxMessages the maximum number of pulled messages
+	 * @param maxMessages the maximum number of pulled messages. If this value is null then there is no limit
```

Comment:
Would it make sense to replace "there is no limit" with "up to `Integer.MAX_VALUE` messages will be requested"? I know that you documented it in refdoc, but for people who read javadoc it would be better to be explicit.

Reply:
Done.